src/pages/create/OfferForm.tsx

```diff
@@ -262,7 +281,10 @@ const OfferForm = () => {
         scopeStepConfiguration,
         typeAndThemeStepConfiguration,
         calendarStepConfiguration,
-        locationStepConfiguration,
+        {
+          ...locationStepConfiguration,
+          offerId,
+        },
```

Comment:
I think this isn't needed. offerId should be passed automatically to all steps in the configurations array

Reply:
Yes you're right :)